external/wakaama/examples/shared/Make.defs

```diff
@@ -38,5 +38,9 @@
 CSRCS += commandline.c connection.c platform.c memtrace.c
 CFLAGS+=-I$(TOPDIR)/../external/wakaama/core
 
+ifeq ($(CONFIG_LWM2M_SECURITY),y)
```

Comment:
Please, check final source codes for external/wakaama/examples/shared/Make.defs.
We already removed LWM2M_SECURITY configuration. Thanks for reviewing.